### ISSUE TITLE
Sometimes doesn't open with inaccessible paths error

### ISSUE THREAD

**Issue by Spiderfffun** [kelly-lin/ranger.nvim#21]
The error quickly flashes in the terminal that opens.
`Inaccessible paths: /example/path/'/example/path`
ranger version: ranger 1.9.3
neovim version: v0.10.0

I might try to see how this works and fix it later, no guarantees tho.

**Comment by Spiderfffun** [comment]
This happens because selectfile is sometimes an absolute path instead of a relative path from what I can tell.
So build_ranger_cmd needs to be changed to account for that. I'm not that good at lua, but will now attempt to make a fix.

**Comment by kelly-lin** [comment]
Hi @Spiderfffun, how do you reproduce this? Do you just open ranger?

**Comment by Spiderfffun** [comment]
> Hi @Spiderfffun, how do you reproduce this? Do you just open ranger?

Yeah, no matter the way I open it if I am not in the same directory it instantly closes.
